HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -315,14 +319,59 @@ wait_server(){
 	echo "server is ready!"
 }
 
+edit_model(){
+    cp -r $1 $TMPDIR/local_model_repo/
+    IFS='/' read -ra ADDR <<< "$1"
+    CONFIG=$TMPDIR/local_model_repo/${ADDR[-1]}/config.pbtxt
+
+    echo "" >> "$CONFIG"
+    echo "instance_group [" >> "$CONFIG"
+    echo "  {" >> "$CONFIG"
+    echo "    count: "$2 >> "$CONFIG"
+    echo "    kind: KIND_CPU" >> "$CONFIG"
+    echo "  }" >> "$CONFIG"
+    echo "]" >> "$CONFIG"
+    echo "" >> "$CONFIG"
+
+    if grep -Fq "onnx" "$CONFIG"
+    then
+	echo "parameters { key: \"intra_op_thread_count\" value: { string_value: \"1\" } }" >> "$CONFIG"
```

Comment:
as above, use cat/heredoc (also avoids need to escape quotes)

Reply:
Switched